app/src/Services/ImageToPdfConversionService.php

```
@@ -327,6 +362,22 @@ private function createSimplePdf(\GdImage $image, int $width, int $height, strin
 
         imagedestroy($resizedImage);
 
+        if (func_num_args() >= 4) {
+            $previewCopy = tempnam(sys_get_temp_dir(), 'waiver_preview_');
+            if ($previewCopy !== false) {
+                $previewCopyJpg = $previewCopy . '.jpg';
+                @rename($previewCopy, $previewCopyJpg);
+                if (@copy($tempJpeg, $previewCopyJpg)) {
+                    $previewPath = $previewCopyJpg;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$previewPath'. (undefined)

(UnusedLocalVariable)